frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -63,18 +63,12 @@ export default function AppNavbar({
                   <Nav.Link as={Link} to="/restaurants">
```

Comment:
NavBar.js must also include changes from main. Consider rebasing off of main to include these changes.